server/src/test/ZetaSqlWrapper.spec.ts

```diff
@@ -24,44 +26,46 @@ describe('ZetaSqlCatalogTest', () => {
     expectedProjectId?: string,
   ): Promise<void> {
     // arrange
-    const zetaSqlCatalog = ZetaSqlCatalog.getInstance();
+    const zetaSqlWrapper = new ZetaSqlWrapper();
 
-    (zetaSqlCatalog as any).catalog.register = async (): Promise<void> => {
+    getCatalog(zetaSqlWrapper).register = async (): Promise<void> => {
       // do nothing
     };
-    zetaSqlCatalog.registerAllLanguageFeatures = async (): Promise<void> => {
+    (zetaSqlWrapper as any).registerAllLanguageFeatures = async (): Promise<void> => {
       // do nothing
     };
 
     // act
     try {
-      await zetaSqlCatalog.register(tableDefinitions);
+      await zetaSqlWrapper.registerCatalog(tableDefinitions);
     } catch (e) {
       console.log(e);
     }
 
     // assert
+    const simpleCatalog = getCatalog(zetaSqlWrapper);
     if (expectedProjectId) {
-      const projects = (zetaSqlCatalog as any).catalog.catalogs;
+      const projects = simpleCatalog.catalogs;
       assertThat(projects.size, 1);
       assertThat(projects.get(expectedProjectId)?.name, expectedProjectId);
     }
 
-    const datasets = expectedProjectId
-      ? (zetaSqlCatalog as any).catalog.catalogs.get(expectedProjectId)?.catalogs
-      : (zetaSqlCatalog as any).catalog.catalogs;
+    const datasets = expectedProjectId ? simpleCatalog.catalogs.get(expectedProjectId)?.catalogs : simpleCatalog.catalogs;
     if (expectedDataSet) {
-      assertThat(datasets?.size, 1);
-      assertThat(datasets?.get(expectedDataSet)?.name, expectedDataSet);
+      assert.ok(datasets);
+      assertThat(datasets.size, 1);
+      assertThat(datasets.get(expectedDataSet)?.name, expectedDataSet);
     }
 
-    const tables = expectedDataSet ? datasets?.get(DATA_SET)?.tables : (zetaSqlCatalog as any).catalog.tables;
-    assertThat(tables?.size, 1);
-    assertThat(tables?.get(table)?.name, table);
+    const tables = expectedDataSet ? datasets?.get(DATA_SET)?.tables : simpleCatalog.tables;
+    assert.ok(tables);
+    assertThat(tables.size, 1);
+    assertThat(tables.get(table)?.name, table);
```

Comment:
Try:
```
assertThat(tables, defined());
assertThat(tables, hasSize(1));
assertThat(tables?.get(table)?.name, table);
```

Reply:
Review others in this file.

Reply:
Unfortunately it doesn't work on compile time, I already tried to find solution
![image](https://user-images.githubusercontent.com/3650304/152991201-2ceff534-34d5-4c84-860c-f54b2974426a.png)
It requires to add more `?` so I decided to use `assert.ok`